pkg/tui/progress/display_props.go

```diff
@@ -78,7 +78,7 @@ func (m Mode) String() string {
 	var s string
 	switch m {
 	case ModeSpinner:
-		s = "spinner"
+		s = "running"
```

Comment:
why, and why not edit other "spinner"

Reply:
because they are really spinning...

Reply:
ok i think the json marshal could also be adjusted